Source/Models/SwishError.swift

```diff
@@ -3,7 +3,8 @@ import Argo
 
 public enum SwishError {
   case ArgoError(Argo.DecodeError)
-  case InvalidJSONResponse(NSError)
+  case InvalidDeserializeError(NSError)
+  case InvalidParseError(NSError)
```

Comment:
The naming here makes it sound like the error is what's invalid. I wonder if there's a better name for these?